packages/payload/src/config/types.ts

```diff
@@ -880,6 +880,25 @@ export type Config = {
      * @default 'all' // The theme can be configured by users
      */
     theme?: 'all' | 'dark' | 'light'
+    /**
+     * Configure timezone related settings for the admin panel.
+     */
+    timezone?: {
+      /**
+       * The default timezone to use for the admin panel.
+       *
+       * @default 'UTC'
+       */
+      defaultTimezone?: string
+      /**
+       * Provide your own list of supported timezones for the admin panel
+       *
+       * Values should be IANA timezone names, eg. `America/New_York`
+       *
+       * We use `@date-fns/tz` to handle timezones
+       */
+      supportedTimezones?: Option[]
```

Comment:
This needs to stricter types to ensure that only IANA timezones are accepted values, not any string. This also should support a function that passes `defaultTimezones` through to make it easier to extend new timezones on top of the default.

Reply:
The list of possible IANA timezones is only available at runtime so instead of types being used here for a possible ~400 strings I'll use config sanitisation and throw an error if an invalid value is provided